SmartEVSE-3/src/evse.cpp

```diff
@@ -584,10 +629,10 @@ void setMode(uint8_t NewMode) {
         if ((Mode != MODE_SOLAR && NewMode == MODE_SOLAR) || (Mode == MODE_SOLAR && NewMode != MODE_SOLAR)) {
             //we are switching from non-solar to solar
             //since we EnableC2 == SOLAR_OFF C2 is turned On now, and should be turned off
-            setAccess(0);                                                       //switch to OFF
+            setAccess(false);                                                       //switch to OFF
```

Comment:
Unnecessary change, distracts from your MQTT code. Please undo.

Reply:
I agree, seems to have been a remnant of me trying to debug why setAccess worked unreliably. I'll put them back how they were.